src/lib/components/ui/input/input.svelte

```
@@ -59,6 +66,7 @@
 			{placeholder}
 			{min}
 			{disabled}
+			on:input={handleChange}
```

ESLint

[error] Using `on:input` to listen to the input event is deprecated. Use the event attribute `oninput` instead(event_directive_deprecated)

(svelte/valid-compile)